addons/refuel/XEH_postInit.sqf

```diff
@@ -1,7 +1,28 @@
 #include "script_component.hpp"
+if (!GVAR(enabled)) exitWith {};
+
+["All", "InitPost", {
```

Comment:
Will it be better to replace `All` with each `cacheRefuelClasses`? It could skip at least `Man` initPost. I don't know XEH well enough.

Reply:
with params index 4 you can add excluded Classes maybe adding CAManBase to that to remove some bulk